modules/UI/videolayout/LargeVideoManager.js

```diff
@@ -160,6 +160,18 @@ export default class LargeVideoManager {
         this.videoContainer.removeResizeListener(
             this._onVideoResolutionUpdate);
 
+        if (getSourceNameSignalingFeatureFlag(APP.store.getState())) {
```

Comment:
This is not to fix the bug. Because destroy() is not called when kicked out. But for completeness.